templates/zerver/help/create-a-stream.md

```diff
@@ -0,0 +1,40 @@
+# Create a stream
+On Zulip, streams are similar to conversation threads or subjects. Streams are located on the left sidebar.
+Streams can be either:
+* **Public streams** - Public streams are for open discussions. All users can subscribe to public streams and discuss there.
+* **Private streams** - Private streams are for confidential discussions and are only visible to users who've been invited to subscribe to it.
+
+## Create a new stream
+In order create a stream on Zulip, organization administrators must have disabled the **Only admins may create streams** option in their **Organization settings**.
+
+To create a stream, locate the cog (![cog](/static/images/help/cog.png)) next to **Streams** in your left sidebar. A tooltip labeled **Subscribe, add, or configure streams** will appear.
+
+![Streams Cog](/static/images/help/streams-1.png)
+
+When you click the cog (![cog](/static/images/help/cog.png)), you'll see an overview of all the streams you've subscribed to. Click the button labeled **Create new stream** to create a stream.
+
+![Streams Overview](/static/images/help/streams-overview.png)
+
+A modal window will appear to let you begin creating your stream.
+
+![Streams Modal](/static/images/help/stream-modal.png)
+
+Enter the title of your stream in the **Stream name** input. If you designate your stream as public, the **Stream name** will be displayed under the Streams on the left sidebar. **Stream name** must be unique to all other streams and have a 60-character limit.
```

Comment:
I would say "and be less than 60 characters long."  The wording makes it sound like you have to create a stream name with a 60-character limit.